docs/reference/ml/df-analytics/apis/put-inference.asciidoc

```diff
@@ -0,0 +1,456 @@
+[role="xpack"]
+[testenv="basic"]
+[[put-inference]]
+=== Create {infer} trained model API
+[subs="attributes"]
+++++
+<titleabbrev>Crerate {infer} trained model</titleabbrev>
+++++
+
+Creates an {infer} trained model.
+
+experimental[]
+
+
+[[ml-put-inference-request]]
+==== {api-request-title}
+
+`PUT _ml/inference/<model_id>`
+
+
+[[ml-put-inference-prereq]]
+==== {api-prereq-title}
+
+If the {es} {security-features} are enabled, you must have the following 
+built-in roles and privileges:
+
+* `machine_learning_admin`
+
+
+* source index: `read`, `view_index_metadata`
```

Comment:
```suggestion
```
There are no additional special roles. Either they must be a cluster admin, or a `machine_learning_admin`